### PR TITLE
Added several applications and games.

### PR DESCRIPTION
**Pull request by ichlaffterlalu**
Added several applications and games tested on vanilla WSA and Magisk-WSA on ASUS ROG Flow X13 (Ryzen 9 5900HS).

OS Features:
- Fingerprint (failed)

Updated status:
- Arcaea with version 3.8.8

New applications:
- AIDE
- CamScanner
- Dcoder
- Duolingo
- GeoGebra Graphing Calc
- Gojek
- Google Meet
- iPusnas (National Library of Indonesia)
- JAKI - Jakarta Kini
- JuiceSSH
- Khan Academy
- KRL Access
- Libby
- LinkedIn
- ManCityApp
- Microsoft Azure
- Mobile JKN
- MOLA
- Moodle
- Network IP Scanner
- QPython 3L
- SATRIA
- Slack
- Symbolab
- Trello
- Vidio
- Zoom

New games:
- Bad Piggies HD
- BanG Dream
- Microsoft Wordament

### CODE REVIEW

**Comment by ichlaffterlalu** [comment]
New applications:
- CamScanner
- Dcoder
- GeoGebra Graphing Calc
- iPusnas (National Library of Indonesia)
- Microsoft Azure
- QPython 3L
- Slack
- Symbolab
- Trello

**Comment by Ratulhasan14789** [comment]
OUT OF TOPIC, one question: how did you use both wsa and magisk (probably the GSM one) at the same time??

File: README.md [hunk 1]
```diff
@@ -59,7 +60,8 @@ wsa://com.android.settings
 | ADM | 12.5.4 | ✅ |||
 | ADM Pro | 6.4.0 | ✅ |||
 | Aegis | 2.0.2 | ✅ |||
-| AFK Arena | 1.72.01 | ⚠️ | Can't login using Google account || 
+| AFK Arena | 1.72.01 | ⚠️ | Can't login using Google account ||
+| AIDE | 3.2.210316 | ✅ | | Might require GMS. |
```
Comment (pratyakshm) [inline README.md:64]:
```suggestion
| AIDE | 3.2.210316 | ✅ | | Might optionally require GMS |
```

File: README.md [hunk 2]
```diff
@@ -79,13 +81,16 @@ wsa://com.android.settings
 | Apple Music | 3.7.1 | ✅ |||
 | App分享 (AppShare) | 2.1.1 (164) | ❌ | Can't login ||
 | Authenticator by Microsoft | 6.2110.6737 |🆖| Requires GMS ||
-| Arcaea |  | ✅ |||
+| Arcaea | 3.8.8 | ⚠️ | Keyboard doesn't work on login/register form. | |
```
Comment (pratyakshm) [inline README.md:84]:
```suggestion
| Arcaea | 3.8.8 | ⚠️ | Keyboard doesn't work on login/register form | |
```

File: README.md [hunk 3]
```diff
@@ -140,34 +150,46 @@ wsa://com.android.settings
...
 | iOS app (any) | | ❌ | Thanks for testing, Brad. |
+| iPusnas | 1.5.1 | ✅ | | |
 | Instagram | 210.0.0.28.71 | ✅ ||
+| JAKI - Jakarta Kini | 1.2.34 | 🆖 | Requires GMS on some features. | |
 | Jetpack Joyride | 1.52.1 (58461800) | ⚠️ | Google Play Games sync doesn't work, otherwise the game functionality is fine ||
 | Jet Car Stunts 2 | 1.0.13 | ❌ | Loads up but orientation and menus are broken ||
 | JioSaavn | 8.2.1 | ✅ |Doesn't support fullscreen and rare crashes but running fine|
 | Joey (Reddit client) | 2.0.0.1 | ✅ ||
 | Joplin | 2.4.3 (2097651) | ✅ ||
+| JuiceSSH | 3.2.2 | ⚠️ | Connecting to SSH server needs multiple tries. | |
```
Comment (pratyakshm) [inline README.md:161]:
```suggestion
| JuiceSSH | 3.2.2 | ⚠️ | Connecting to SSH server needs multiple tries | |
```

File: README.md [hunk 4]
```diff
@@ -140,34 +150,46 @@ wsa://com.android.settings
...
 | Kahoot | | ✅ |||
+| Khan Academy | 7.3.3 | ✅ | | |
 | Kik | 7.10.1.176 (82)  | ✅ ||
 | Kindle | 8.47.1.3370 | ✅ | | |
 | KINGDOM HEARTS Uχ Dark Road | 4.4.0 (Offline) | ✅ | GMS warnings might appear but these can be safely ignored ||
 | Konosuba:FD | 1.12.1 | 🆖 | Requires GMS ||
+| KRL Access | 4.1.0 | ❌ | App crashes immedately. | |
```
Comment (pratyakshm) [inline README.md:168]:
```suggestion
| KRL Access | 4.1.0 | ❌ | App crashes | |
```

File: README.md [hunk 5]
```diff
@@ -272,4 +302,5 @@ wsa://com.android.settings
 | Yandex.Maps | 10.6.0 | ⚠️ | Map doesn't work |
 | Ymusic | 3.7.2 | ✅ ||
 | ZArchiver | 0.9.5.8 (9596) | ✅ ||
-| Zenly (Without Google Services) | 4.55.2 | ⚠️ | App crashes after login, but background location works | 
+| Zenly (Without Google Services) | 4.55.2 | ⚠️ | App crashes after login, but background location works |
```
Comment (pratyakshm) [inline README.md:305]:
```suggestion
| Zenly (w/o GMS) | 4.55.2 | ⚠️ | App crashes after login, but background location works |
```

**Review by pratyakshm — CHANGES_REQUESTED** [review]

**Comment by pratyakshm** [comment]
Thank you so much!

**Comment by ichlaffterlalu** [comment]
> OUT OF TOPIC, one question: how did you use both wsa and magisk (probably the GSM one) at the same time??
@Ratulhassan14789 simply use one version at a time: install WSA-Magisk, then uninstall, then install vanilla WSA :D

**Pull request merged**